test/unit/userAgentDataConsts.ts

```diff
@@ -27,6 +27,8 @@ const AGENT_DATA_LIST = [
                 version: "87",
                 chromium: true,
                 webkit: false,
+                chromiumVersion: "87",
```

Comment:
nit)
If I check only the major version, does the type need to be string?

Reply:
@mixed 

In that case, I think you should use parseInt.

Reply:
@daybrush 
The type of the return value is slightly different from what I thought. What is the difference between majorVersion and version? The expected value is that majorVersion represents major version, and I thought it would be a numeric type, and version represents full version, and I thought it was a string type. From the description of the document, it seems that the only difference between majorVersion and version is the type. Is that right?

![image](https://user-images.githubusercontent.com/22300/108963645-b99e7c00-76bd-11eb-92a9-0b7f2193988f.png)

Reply:
@mixed 

Unfortunately, userAgentData does not indicate the minor version of the engine.

Reply:
So, is there no need to change the description? :)

Reply:
@mixed 
Oh thank you.